packages/create-astro/README.md

```diff
@@ -45,16 +45,19 @@ npm create astro@latest my-astro-project -- --template cassidoo/shopify-react-as
 
 May be provided in place of prompts
 
-| Name                         | Description                                            |
-| :--------------------------- | :----------------------------------------------------- |
-| `--template <name>`          | Specify your template.                                 |
-| `--install` / `--no-install` | Install dependencies (or not).                         |
-| `--git` / `--no-git`         | Initialize git repo (or not).                          |
-| `--yes` (`-y`)               | Skip all prompt by accepting defaults.                 |
-| `--no` (`-n`)                | Skip all prompt by declining defaults.                 |
-| `--dry-run`                  | Walk through steps without executing.                  |
-| `--skip-houston`             | Skip Houston animation.                                |
-| `--typescript <option>`      | TypeScript option: `strict` / `strictest` / `relaxed`. |
+| Name                       | Description                                                |
+| :------------------------- | :--------------------------------------------------------- |
+| `--help (-h)`              | Display available flags.                                   |
+| `--template <name>`        | Specify your template.                                     |
+| `--install / --no-install` | Install dependencies (or not).                             |
+| `--git / --no-git`         | Initialize git repo (or not).                              |
+| `--yes (-y)`               | Skip all prompts by accepting defaults.                    |
+| `--no (-n)`                | Skip all prompts by declining defaults.                    |
```

Comment:
```suggestion
| `--install` / `--no-install` | Install dependencies (or not).                             |
| `--git` / `--no-git`         | Initialize git repo (or not).                              |
| `--yes`  (`-y`)              | Skip all prompts by accepting defaults.                    |
| `--no`  (`-n`)               | Skip all prompts by declining defaults.                    |
```
I'm confused by the formatting change here. As these are two separate options, shouldn't they be indicated separately as they were before? Was this wrong before?